dynamixel_port:
  port_name: "/dev/input/dx_upper"
  baud_rate: 1000000
  joints:
    - body_roll_joint

  body_roll_joint:            {id: 2, center: 2048, home: 0.0, mode: 3, vel: 200, acc: 0, pos_d_gain: 0, pos_i_gain: 0, pos_p_gain: 800, gear_ratio: 1}